ui/sign_in_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+describe 'Sign in page' do
+  context 'when unauthenticated' do
+    include_context 'as guest'
+
+    before do
+      ui.sign_in.load
+    end
+
+    it 'Warning is displayed when the user clicks login with blank username' do
+      ui.sign_in.with_email_and_password('', '')
+      expect(ui.sign_in).to have_warning_title
+    end
+
+    it 'Warning is displayed when the user clicks login with invalid username' do
+      ui.sign_in.with_email_and_password(nil, 'Strongassword')
+      expect(ui.sign_in).to have_warning_title
+    end
+
+    it 'is shown expanded Help menu' do
+      sleep(10)
+      ui.sign_in.problem_signing_in.click
+      expect(ui.sign_in).to have_content 'I have forgotten my password'
```

Comment:
This is an element in the page model. Is it better to define all key elements there and then not need to have strings here in the spec?